src/components/about/sections/PeopleDisplay.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import "../index.css";
+import PersonDisplay from "./PersonDisplay";
+import { Row } from "react-bootstrap";
+
+// i18n translations might still be loaded by the http backend
+// use react's Suspense
+
+export default function peopleDisplay() {
+  const peopleData = [
```

Comment:
the variable names here need work

first, this is not quite peopleData, that would be inaccurate. it would be more along, `peopleOrdering`

second, even though the key happens to be the name, that's not quite future proof. so we shouldn't call it `name`. instead, i would call it `translationKey`, because that's what it is: all it does is eventually go to i18n